internal/pkg/service/common/etcdop/key.go

```diff
@@ -171,7 +173,36 @@ func (v KeyT[T]) GetKV(client etcd.KV, opts ...etcd.OpOption) op.WithResult[*op.
 	)
 }
 
-func (v KeyT[T]) Get(client etcd.KV, opts ...etcd.OpOption) op.WithResult[T] {
+// GetOrNil gets a decoded value of the key as pointer.
+// If the key is missing, result is nil pointer.
+func (v KeyT[T]) GetOrNil(client etcd.KV, opts ...etcd.OpOption) op.WithResult[*T] {
+	return op.NewForType(
+		client,
```

Comment:
After the refactoring, we have following get methods:
- `GetOrNil` - `op.WithResult[*T]`
- `GetOrErr` - `op.WithResult[T]`
- `GetOrEmpty` - `op.WithResult[T]`
- `GetKV` - `op.WithResult[*op.KeyValueT[T]]`

So you immediately know how it works from the code. 

For example, `GetOrErr` is used in the definition repository, because the missing entity is a problem.

For statistics, `GetOrEmpty` makes sense.